protocol/common/shard.go

```diff
@@ -43,5 +48,15 @@ func (s *Shard) Protobuffer() *protobuf.Shard {
 	}
 }
 
-const MainStatusShard = 16
-const UndefinedShardValue = 0
+func DefaultNonProtectedPubsubTopic(shard *Shard) string {
```

Comment:
Not really important - but do we plan to extract some data from `shard` variable in the future? Otherwise we only care whether its `nil` or not.

Reply:
for now we're only interested in it being nil or not!